package.json

```diff
@@ -21,14 +23,19 @@
     "e2e:cy:run:regression": "yarn --cwd ./projects/storefrontapp-e2e-cypress run cy:run:regression",
     "e2e:cy:run:regression:ci": "yarn --cwd ./projects/storefrontapp-e2e-cypress run cy:run:regression",
     "e2e:cy:run:all:ci": "yarn --cwd ./projects/storefrontapp-e2e-cypress run cy:run:all",
+    "e2e:cy:run:vendor": "yarn --cwd ./projects/storefrontapp-e2e-cypress run cy:run:vendor",
+    "e2e:cy:run:vendor:cds": "yarn --cwd ./projects/storefrontapp-e2e-cypress run cy:run:vendor:cds",
```

Comment:
In order to keep it consistent, it's better to put `cds` before `vendor`, so the script name should be: `e2e:cy:run:cds:vendor`

Reply:
I think it would be e2e:cy:cds:run:vendor